keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1946,6 +1947,16 @@ in back-to-back releases.
 - Feature implemented behind a feature flag
 - Ensure proper tests are in place.
 
+#### Beta
```

Comment:
While we're making an edit to the KEP, @liggitt had also pointed out that webhooks perform the minimal number of conversions. Let's add this to the list:

Perform minimal possible number of conversions when evaluating multiple admission policies for a request resource. If multiple admission policies require the same conversion, convert only once. From @liggitt: "webhook code loops up one level, first accumulates all the validation webhooks we'll run, then converts to the versions needed by those webhooks then evaluates in parallel"

Reply:
Added.